tests/unit/test_general_options.py

```diff
@@ -229,8 +229,7 @@ class       Command group for CIM classes.
      {'general': ['-s', 'httpx://blah'],
       'cmdgrp': 'class',
       'args': ['get', 'blah']},
-     {'stderr': ['Error: Invalid scheme on server argument. httpx://blah Use '
-                 '"http" or "https"'],
+     {'stderr': ["Error: Unsupported scheme 'httpx' in URL 'httpx://blah"],
```

Comment:
Is this change done in the code?

Reply:
There was a change in the code somewhere that causes this.  The message is from pywbem.  Something got dropped somewhere because the pywbemcli _validate_url is no longer being called and the exception from pywbem is what you see.  Somewhere in the move to connect we dropped the PywbemServer. server setter and that is where the call to the _validate_url is.  I will sort that out this afternoon